### PR TITLE
Added three horrible puns

### PR DESCRIPTION
**Pull request by trbst**

### CODE REVIEW

File: puns.json [hunk 1]
```diff
@@ -158,5 +158,17 @@
...
     "pun": "Q: Why did the functions stop calling each other?",
     "punchline": "A: Because they had constant arguments!"
+  },
+  {
+    "pun":"Q: Whats the most cutting edge language?",
+    "punchline": "A: C#."
+  },
+  {
+    "pun":"Q: Why couldn't the programmer dance to the song?",
+    "punchline": "A: Because he didn't get the algo-rhythm!"
```
Comment (AlexLakatos) [inline puns.json]:
```suggestion
    "punchline": "A: Because they didn't get the algo-rhythm!"
```

Comment (AlexLakatos) [reply puns.json]: I'm trying to keep these gender-neutral, is it ok if we change the punchline a bit?

Comment (trbst) [reply puns.json]: Of course, I didn't pay attention to it!

**Comment by AlexLakatos** [comment]
Thanks for the PR. Happy Hacktoberfest!

**Pull request merged**